bench-tps/src/cli.rs

```diff
@@ -382,6 +385,23 @@ pub fn build_args<'a>(version: &'_ str) -> App<'a, '_> {
                 .validator(is_keypair)
                 .help("File containing the node identity (keypair) of a validator with active stake. This allows communicating with network using staked connection"),
         )
+        .arg(
+            Arg::with_name("load_accounts_from_address_lookup_table")
+                .long("load-accounts-from-address-lookup-table")
```

Comment:
not a huge fan of the name as it's not obvious (without reading the help message) we're passing the sbf program id, not the lookup table id.

It's also odd imo to have 2 fields in a single argument. Why not make 2 args that require eachother: `--write-program-pubkey` and `--write-program-num-loaded-accounts`

Reply:
Initially tried of having two args that require each other, that didn't work. But I do want these two args be a bundle, so did it this way.